src/examples/get_available_disbursement_banks.php

```diff
@@ -0,0 +1,12 @@
+<?php
+    require('config/xendit_php_client_config.php');
+    require('XenditPHPClient.php');
+    
+    $options['secret_api_key'] = constant('SECRET_API_KEY');
+    $options['server_domain'] = constant('SERVER_DOMAIN');
```

Comment:
If that's the case.

we should change this
```
        function __construct ($options) {
            if ($options['server_domain']) {
                $this->server_domain = $options['server_domain'];
            } else {
                $this->server_domain = 'https://api.xendit.co';
            }

            $this->secret_api_key = $options['secret_api_key'];
        }
```

into this
```
        function __construct ($options) {
            $this->server_domain = 'https://api.xendit.co';
            $this->secret_api_key = $options['secret_api_key'];
        }
```

What do you think?